easybuild/easyconfigs/p/pysamstats/pysamstats-1.1.2-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = "PythonPackage"
+
+name = 'pysamstats'
+version = '1.1.2'
+
+homepage = 'https://github.com/alimanfoo/pysamstats'
+description = """A Python utility for calculating statistics
+against genome positions based on sequence alignments from a SAM or BAM file."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['6d8b41934fdc700552d689f473860c64525a88600460344d0cac2095424750c6']
+
+use_pip = True
+download_dep_fail = True
+
+preinstallopts = "sed -i -e 's/ (<0.16)//g' setup.py && "
```

Comment:
Add a comment above this line to explain it?

Reply:
done